### PR TITLE
Arn 299 add guillotine event

### PR DESCRIPTION
**Pull request by adelaidaa**
Use test containers to automate the Integration testing against AWS LocalStack on the pipeline and to be able to run the test locally without manually running docker and the scripts to generate the testing queues setup

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -6,18 +6,8 @@ orbs:
...
-          - DEBUG=${DEBUG- }
-          - DATA_DIR=/tmp/localstack/data
-          - DOCKER_HOST=unix:///var/run/docker.sock
-          - LAMBDA_EXECUTOR=${LAMBDA_EXECUTOR- }
-          - AWS_EXECUTION_ENV=True
-          - DEFAULT_REGION=eu-west-2
-          - TMPDIR=/private
+    machine:
+      image: ubuntu-2004:202101-01
     environment:
```
Comment (adelaidaa) [inline .circleci/config.yml:11]: we seem to need to use machine executor for testcontainers to work - https://www.testcontainers.org/supported_docker_environment/continuous_integration/circle_ci/

File: .circleci/config.yml [hunk 2]
```diff
@@ -28,18 +18,6 @@ jobs:
     executor: validator
     steps:
       - checkout
-      - hmpps/install_aws_cli
```
Comment (adelaidaa) [inline .circleci/config.yml:31]: https://discuss.circleci.com/t/linux-machine-executor-images-2021-january-q1-update/38976

The machine we are using  ubuntu-2004:202101-01 has AWS CLI v2 pre-installed

File: .circleci/config.yml [hunk 3]
```diff
@@ -28,18 +18,6 @@ jobs:
...
-          name: install dockerize
-          command: wget https://github.com/jwilder/dockerize/releases/download/$DOCKERIZE_VERSION/dockerize-linux-amd64-$DOCKERIZE_VERSION.tar.gz && sudo tar -C /usr/local/bin -xzvf dockerize-linux-amd64-$DOCKERIZE_VERSION.tar.gz && rm dockerize-linux-amd64-$DOCKERIZE_VERSION.tar.gz
-          environment:
-            DOCKERIZE_VERSION: v0.3.0
-      - run:
-          name: Wait for localstack
-          command: dockerize -wait tcp://localhost:4575 -timeout 2m
-      - run:
-          name: Set up topic
-          command: src/test/resources/localstack/setup-sns.sh
```
Comment (adelaidaa) [inline .circleci/config.yml:42]: we don't need to do this in the pipeline as localstack is now configured in the Integration testing phase

**Review by wood-n — APPROVED** [review]
I like this, it feels more sophisticated and self-contained 👍

**Review by SWhitlam — APPROVED** [review]

**Pull request merged**